antenna/breakpad_resource.py

```diff
@@ -150,10 +151,10 @@ def check_health(self, state):
 
     def hb_health_stats(self):
         # The number of crash reports sitting in the queue
-        self.mymetrics.gauge('save_queue_size', len(self.save_queue))
+        self.mymetrics.gauge('save_queue_size', len(self.crashmover_save_queue))
 
         # The number of actively running coroutines saving crashes
-        self.mymetrics.gauge('active_save_workers', len(self.pipeline_pool))
+        self.mymetrics.gauge('active_save_workers', len(self.crashmover_pool))
```

Comment:
Given that this is either 0 or 1, I'm not sure we need this anymore. But since it's sent during heartbeats, might as well keep it.